include/battle.h

```diff
@@ -139,14 +139,14 @@ struct ProtectStruct
 
 struct SpecialStatus
 {
-    u8 statLowered:1;
-    u8 lightningRodRedirected:1;
-    u8 restoredBattlerSprite: 1;
-    u8 intimidatedMon:1;
-    u8 traced:1;
-    u8 ppNotAffectedByPressure:1;
-    u8 flag40:1;
-    u8 focusBanded:1;
+    u32 statLowered:1;
+    u32 lightningRodRedirected:1;
+    u32 restoredBattlerSprite: 1;
+    u32 intimidatedMon:1;
+    u32 traced:1;
+    u32 ppNotAffectedByPressure:1;
+    u32 flag40:1;
+    u32 focusBanded:1;
     u8 field1[3];
```

Comment:
Is this padding needed?

Reply:
Yes. The padding is needed for the struct to align.

Reply:
Unless the struct has `__attribute__((packed))`, filler bytes are not needed. s32 is implicitly aligned to word.

Reply:
Oops. Forgot it wasn't packed. Yeah, it OK's without filler. :)